change/@azure-msal-browser-05d052ac-925b-40f2-8d36-78bcab460285.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "- Log number of accounts in trace mode. - Validate that `getActiveAccount()` picks up legacy account identifier from local storage. #5525",
```

Comment:
```suggestion
  "comment": "Log number of accounts in trace mode. #5525",
```

Reply:
I don't see any changes to the lib besides some logger changes?

Reply:
There is also a new unit test here https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5525/files#diff-f936803be34376b4fc36745b78ec5110b6e3b420d907b033fa6924453d2815d5R3235 to make sure old account ids from local cache are honored.

Reply:
Gotcha. Since this will be published in our release notes we should stick to just what has changed in the lib